src/components/common/page/Page.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, ReactNode } from 'react';
+import { Box } from '@mui/material';
+import styles from './Page.module.scss';
+import Header from '@/components/common/header';
+interface PageProps {
+  children?: ReactNode;
+}
+const Page: FC<PageProps> = ({ children }) => {
```

Comment:
In my opinion, it is better to add a space between `PageProps` and `Page` because that is what you do in subsequent comoponents

Reply:
fixed